src/marketing/product-compare.md

```diff
@@ -11,11 +11,15 @@ Unlike the [Recently Viewed / Compared Products ]({% link marketing/products-vie
 
 ## Compare products on the storefront
 
+There are a few ways navigate to comparison list on the storefront.
+
+From the catalog pages:
```

Comment:
Let's make theses subhead so that they are displayed in the page nav on the right

```suggestion
### From catalog pages
```